lib/rihanna/job.ex

```diff
@@ -363,4 +359,19 @@ defmodule Rihanna.Job do
       job_module.after_error(reason, arg)
     end
   end
+
+  # Some operations can use the shared database connection as they don't use locks
```

Comment:
I believe you mean "can't"

Reply:
Can is right, by shared I mean the one that can optionally be the repo. Should I reword this?

Reply:
Ah, I misread it. All good!